Tools/toggle_bitcoin_taproot_bip8_lottrue..sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+if [[ "$0" = "$BASH_SOURCE" ]]; then
+    echo "This script must be sourced \". toggle_bitcoin_taproot_bip8_lottrue.sh\"" 
+    exit 1
+fi
+
+set -e
+
+if [[ "$BTCPAYGEN_ADDITIONAL_FRAGMENTS" =~ "bitcoin-taproot-bip8" ]]; then
+ read -p "The unofficial taproot node release is already active. Type 'official' to change back to the official release `echo $'\n> '`" yn
```

Comment:
There is no official release. ST is being deceptively branded "Bitcoin Core" without qualifiers, but it is not following the standard release norms.

Reply:
I think using `bitcoin_core` instead of `official` would be better.